deepchem/models/torch_models/layers.py

```diff
@@ -6477,3 +6477,51 @@ def forward(self, inputs: List[np.ndarray]):
         if self.activation_fn is not None:
             mol_features = self.activation_fn(mol_features)
         return mol_features
+
+
+class Logistic(nn.Module):
+    """A logistic function or logistic curve is a common S-shaped curve (sigmoid curve).
+    Logistic functions are often used in artificial neural networks to introduce
+    nonlinearity in the model or to clamp signals to within a specified interval.
```

Comment:
Can you add an example showing how a user can have a logistic layer output in a different interval than the normal (0,1)?

Reply:
😀 thanks for suggesting, will add it.